config/initializers/test_track_generate_timestamp.rb

```diff
@@ -0,0 +1,9 @@
+BUILD_TIMESTAMP_FILE_PATH = 'testtrack/build_timestamp'.freeze
+
+if Rails.env.test? || Rails.env.development?
```

Comment:
Getting some tests around this behavior would be great too. Not sure if you can test an initializer. But we could pull this logic into a function, test that and then just call that here